apps/transmitmedia.cpp

```diff
@@ -829,33 +831,37 @@ class UdpCommon
             Error(SysError(), "UdpCommon::Setup: ioctl FIONBIO");
         }
 
-        sadr = CreateAddr(host, port);
+        interface_addr  = sockaddr_any(AF_INET);
+        interface_addr.sin.sin_family      = AF_INET;
+        interface_addr.sin.sin_addr.s_addr = htonl(INADDR_ANY);
+        interface_addr.sin.sin_port        = htons(port);
```

Comment:
You can use the API of `sockaddr_any` directly, or better use `CreateAddr` as you already have it.
```suggestion
    interface_addr = CreateAddr("", port, AF_INET);
```